qa/rolling-upgrade/src/test/resources/processor/UploadModelRequestBody.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "traced_small_model",
+  "version": "1.0.0",
+  "model_format": "TORCH_SCRIPT",
+  "model_task_type": "text_embedding",
+  "model_content_hash_value": "e13b74006290a9d0f58c1376f9629d4ebc05a0f9385f40db837452b167ae9021",
+  "model_group_id": "<MODEL_GROUP_ID>",
```

Comment:
Should this be "%s" instead of "<MODEL_GROUP)ID>"?

Reply:
In Integ test similar kind of work is done. If this has to change then I can raise a new PR for that. Already this PR became heavy. Probably in next iteration for multimodal test I will add it. Does it work for now?